components/provider/src/lib.rs

```diff
@@ -67,7 +67,7 @@
 mod cloneable_any;
 mod data_entry;
 #[macro_use]
-mod data_key;
+pub mod data_key;
```

Comment:
You won't need this change after you rebase to master.

Reply:
Done.